graphdatascience/graph/graph_sample_runner.py

```diff
@@ -27,3 +27,20 @@ def rwr(self, graph_name: str, from_G: Graph, **config: Any) -> Tuple[Graph, "Se
         result = self._query_runner.run_query_with_logging(query, params).squeeze()
 
         return Graph(graph_name, self._query_runner, self._server_version), result
+
+    @client_only_endpoint("gds.alpha.graph.sample")
+    @compatible_with("construct", min_inclusive=ServerVersion(2, 2, 0))
```

Comment:
This should be `@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 4, 0))` since GDS 2.4.0 is the version this algorithm is being added